lib/ex_machina.ex

```diff
@@ -58,6 +58,17 @@ defmodule ExMachina do
       def create_list(number_of_factories, factory_name, attrs \\ %{}) do
         ExMachina.create_list(__MODULE__, number_of_factories, factory_name, attrs)
       end
+
+      def factory(factory_name, attrs \\ [])
```

Comment:
Is this needed? I thought we already had a catch all defined in `before_compile`?